src/main/resources/schema/type.graphqls

```diff
@@ -1,13 +1,58 @@
+"Custom Scalar Date"
+scalar DateTime
+
+
 type Wish {
- id: ID!
- authorId: ID!
- title: String!
- content: String!
- url: String!
- coverImageUrl: String
+  id: ID!
+  author: Wisher!
+  title: String!
+  description: String!
+  tags: String!
+  coverImageUrl: String
+  votes: [Vote]
```

Comment:
Should not return null - empty array would probably work better

Reply:
Resolved :ok_hand: